backend/.env

```diff
@@ -0,0 +1,8 @@
+PORT = 5000;
```

Comment:
Add `.env` file to gitignore, and delete from here. It should not be available on Github since it contains sensitive details.